packages/trader/src/Stores/Modules/Trading/Helpers/turbos-utils.js

```diff
@@ -6,6 +6,7 @@ export const getTurbosColor = type => {
         case 'TURBOSSHORT':
             return BARRIER_COLORS.RED;
         case 'TURBOSLONG':
+            return BARRIER_COLORS.GREEN;
```

Comment:
```suggestion
```
@kate-deriv it's gray for Long as well as for a default case )

Reply:
We have a conclusion after call: all barriers are gray and solid by default. When user hover, Long become green + dashed and Short become Red + dashed.